packages/metal-soy/src/Soy.js

```diff
@@ -40,6 +40,8 @@ class Soy extends IncrementalDomRenderer.constructor {
 	 * template call's data. The copying needs to be done because, if the component
 	 * itself is passed directly, some problems occur when soy tries to merge it
 	 * with other data, due to property getters and setters. This is safer.
+	 * Also calls the component's "getTemplateData" is called to let it change the
```

Comment:
Sorry, wrote it while listening to the training so I probably wrote it in two phases…

Gosh I made a conflict merge artifact without using git…